yt_dlp/extractor/alphaporno.py

```diff
@@ -1,75 +1,142 @@
+import re
+
 from .common import InfoExtractor
 from ..utils import (
     int_or_none,
     parse_duration,
     parse_filesize,
-    parse_iso8601,
+    parse_resolution,
+    unified_timestamp,
+    urljoin,
 )
 
 
 class AlphaPornoIE(InfoExtractor):
     _VALID_URL = r'https?://(?:www\.)?alphaporno\.com/videos/(?P<id>[^/]+)'
-    _TEST = {
-        'url': 'http://www.alphaporno.com/videos/sensual-striptease-porn-with-samantha-alexandra/',
-        'md5': 'feb6d3bba8848cd54467a87ad34bd38e',
-        'info_dict': {
-            'id': '258807',
-            'display_id': 'sensual-striptease-porn-with-samantha-alexandra',
-            'ext': 'mp4',
-            'title': 'Sensual striptease porn with Samantha Alexandra',
-            'thumbnail': r're:https?://.*\.jpg$',
-            'timestamp': 1418694611,
-            'upload_date': '20141216',
-            'duration': 387,
-            'filesize_approx': 54120000,
-            'tbr': 1145,
-            'categories': list,
-            'age_limit': 18,
+    _TESTS = [
+        {
+            'url': 'http://www.alphaporno.com/videos/sensual-striptease-porn-with-samantha-alexandra/',
+            'md5': '7e6a1cdd48fa67362a5a11d7039164e7',
+            'info_dict': {
+                'id': '258807',
+                'display_id': 'sensual-striptease-porn-with-samantha-alexandra',
+                'ext': 'mp4',
+                'title': 'Sensual striptease porn with Samantha Alexandra',
+                'description': 'md5:3c6d31008980654acaeb11451454a62c',
+                'thumbnail': r're:https?://.*\.jpg$',
+                'timestamp': 1418701811,
+                'upload_date': '20141216',
+                'duration': 387,
+                'categories': list,
+                'age_limit': 18,
+            },
+        },
+        {
```

Comment:
can we reformat+dedent all of those tests? the download test script outputs info dicts with indentation intended for `_TESTS` that are formatted like this:
```suggestion
    _TESTS = [{
        'url': 'http://www.alphaporno.com/videos/sensual-striptease-porn-with-samantha-alexandra/',
        'md5': '7e6a1cdd48fa67362a5a11d7039164e7',
        'info_dict': {
            'id': '258807',
            'display_id': 'sensual-striptease-porn-with-samantha-alexandra',
            'ext': 'mp4',
            'title': 'Sensual striptease porn with Samantha Alexandra',
            'description': 'md5:3c6d31008980654acaeb11451454a62c',
            'thumbnail': r're:https?://.*\.jpg$',
            'timestamp': 1418701811,
            'upload_date': '20141216',
            'duration': 387,
            'categories': list,
            'age_limit': 18,
        },
    }, {
```